src/vehicle/rdw.py

```diff
@@ -17,6 +17,8 @@
 from .exceptions import RDWConnectionError, RDWError, RDWUnknownLicensePlateError
 from .models import Vehicle
 
+VERSION = metadata.version(__package__)
```

Comment:
Let's remove the agent completely (as I have done in my other packages as well).